.travis.yml

```diff
@@ -6,8 +6,9 @@ cache:
   directories:
   - $HOME/.m2
 before_install:
-  - pyenv install 3.7.10
-  - pyenv global 3.7.10
+#  - pyenv install --force 3.7.1
```

Comment:
This confuses me:  The description says, "update to 3.7.1" but the change seems to downgrade to a lower version. Is that the intention?

PS Unless you need to keep it for some reason, please remove commented-out code.

Reply:
I've updated PR comment with better comment